examples/hf_serialization.py

```diff
@@ -39,15 +61,43 @@
 logger.addHandler(fh)
 
 
+def check_file_exists(
+    file: str,
+):
+    """
+    Check if file exists and is not empty. If the file is found locally,
+    it is checked for emptiness with `os.path.exists`. If the file is found
+    on S3, it is checked for emptiness by reading the first byte of the file.
+
+    Args:
+        file: The path to check for existence and emptiness. This can
+           either be a local path or an S3 URI.
+    """
+    if os.path.exists(file):
+        return True
+    else:
+        try:
+            with stream_io.open_stream(
+                file,
+                "rb",
+                *s3_read_credentials,
+            ) as f:
+                f.read(1)
+                return True
+        except OSError as e:
+            return False
```

Comment:
Tensorizer should technically return an empty string when reading from a file that is extant-yet-empty to behave like a file-like object. It doesn't at the moment—instead it raises an `OSError`—but if the implementation changes or this somehow manages to open a local file, `bool(f.read(1))` (equivalent to `len(f.read(1)) > 0`) would technically be more correct.
Also the error identifier `e` isn't used in the `except` clause below, so it doesn't need to be captured.
```suggestion
                return bool(f.read(1))
        except OSError:
            return False
```

Reply:
Okay, understood. Thanks for pointing out a forward-compatible change! Committing this.